### DISCUSSION TITLE
Extend Umbraco Cloud API

### DISCUSSION THREAD

**Discussion by lukehook** [umbraco/Umbraco.Cloud.Issues#796] [Ideas]
In the current Cloud API documentation we have POST methods to create a new project which is great and something we're looking to utilise.

It would be great if this could be extended to GET some information from our account. For instance a GetProjects method or similar that would return basic information about a project. Alias, url, projectId and whether it's utilising a baseline for instance would be really useful.

This way we could use the deploy webhook to programatically loop through and process some action on a series of our projects post deployment

**Comment by nul800sebastiaan** [comment]
Hi @lukehook - while going through the older issues on this tracker I noticed this one. 

We currently don't have immediate plans to work on the API features but I've put this on the list of things we will use for inspiration when we're working on updates for the API.

**Comment by lukehook** [comment]
Thanks @nul800sebastiaan that sounds great

**Comment by bjarnef** [comment]
I was also wondering about if there are any REST APIs available for Umbraco Cloud projects.
E.g. fetch information about projects within an organization.

As there number of Umbraco Cloud projects grow, we would like if we internal could create a dashboard or similar to show if projects are up-to-date, on EOL version etc.

Perhaps Umbraco could even return some schema/meta data if the project is is on STS/LTS/EOL:
https://umbraco.com/products/knowledge-center/long-term-support-and-end-of-life/

Of course we could manually map version with these info internal, but would be nicer something similar to what @prjseal did here: https://psw.codeshare.co.uk/

![image](https://github.com/user-attachments/assets/a0b99f3f-8db8-4514-a6e4-9c5a7e6f0794)
